.travis.yml

```diff
@@ -2,14 +2,14 @@ dist: trusty
 language: node_js
 cache: yarn
 node_js:
-- '6.11.5'
```

Comment:
I think we might have to keep testing against node 6 for now, given the non-Neutrino consumers of webpack-chain (and that it would be good to release any new stringify features as a minor version if possible, to make it easier to compare Neutrino 8 and Neutrino 9 configs).

Reply:
Ah good point. I'll revert.

Reply:
I'll drop Node.js 7 though.